src/util/graph.creator.ts

```diff
@@ -43,7 +43,7 @@ export class GraphCreator {
     private static searchRawImportLocations(preparedFileContent: string): string[] {
         return preparedFileContent
             .split('\n')
-            .filter((line: string) => /[ ]from[ ]['"].*['"]/.test(line))
+            .filter((line: string) => /[ ]from[ ]['"].*['"]/.test(line)) // import begins with .
```

Comment:
Not sure if this comment is correct here, I think we still catch "all" imports here, no? E.g. all the packages, ".json"-imports etc... all the stuff we filter out later

Reply:
That's definitely wrong